__types.lua

```diff
@@ -26,6 +26,7 @@
 ---@field renderDistance number
 ---@field activeDistance number
 ---@field currentDistance number
+---@field sprite? { dict: string, txt: string, color: vector4 }
```

Comment:
we should mark the contents of sprite as optional too, just to show that they could update only the color if wanted

Reply:
I actually had this written on my local branch and completely missed commiting the update, but I've also now added optional `dict` and `txt` parameters.